src/server/boot/03-audit.js

```diff
@@ -0,0 +1,41 @@
+import optionalBasicAuth from '../middleware/optional-basic-auth'
+import { models } from '../models'
+import { audit } from '../util/audit'
+
+export default function (app) {
+  app.get(
+    '/api/audit-events',
+    optionalBasicAuth(),
+    app.requirePermission('view audit log'),
+    app.wrap(async (req, res) => {
+      const filter = JSON.parse(req.query.filter || '{}')
+
+      const where = filter.where || {}
+      const limit = +filter.limit || 250
+      const offset = +filter.skip || undefined
+      const order = filter.order
+        ? filter.order.split(' ')
+        : ['timestamp', 'DESC']
+
+      const events = await models.AuditEvent.findAll({
+        include: [
+          { model: models.AuditClientData, as: 'clientData' },
+          { model: models.User },
+        ],
+        where,
+        limit,
+        offset,
+        order: [order],
+      })
+
+      await audit({
```

Comment:
maybe the creation of the audit event should be before the `findAll`, so it would show up in the listing

Reply:
Done